health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/DrawerFieldComposer.js

```
@@ -9,17 +9,19 @@
-  if (panelItem?.showFieldOnToggle && drawerState?.[panelItem.label]) {
+  if (panelItem?.showFieldOnToggle && anyCheck) {
     return panelItem?.conditionalField;
   }
 };
 
-const disableFieldForMandatory = (drawerState,label) => {   // todo need to think about it @nabeel & @jagan
+const disableFieldForMandatory = (drawerState, label) => {
+  // todo need to think about it @nabeel & @jagan
   if (drawerState?.Mandatory && !drawerState?.deleteFlag) {
-    return label=="Mandatory"? true:false;
+    return label == "Mandatory" ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)